src/modules/market/components/market-positions-list--order/market-positions-list--order.jsx

```diff
@@ -14,8 +14,9 @@ export default class Order extends Component {
   static propTypes = {
     isExtendedDisplay: PropTypes.bool,
     isMobile: PropTypes.bool,
+    name: PropTypes.string,
```

Comment:
Maybe this should be called something like `outcomeName` to make it a little clearer what it represents?